src/snapred/backend/recipe/algorithm/FitMultiplePeaksAlgorithm.py

```diff
@@ -67,62 +61,48 @@ def listToWorkspace(self, aList, name):
         ws = WorkspaceFactory.create("Workspace2D", NVectors=1, XLength=len(aList), YLength=len(aList))
         ws.setX(0, np.asarray(aList))
         # register ws in mtd
-        mtd.add(name, ws)
+        mtd.addOrReplace(name, ws)
         return ws
 
     def validateInputs(self) -> Dict[str, str]:
         errors = {}
-        waysToGetPeaks = ["DetectorPeaks", "DetectorPeakIngredients"]
-        definedWaysToGetPeaks = [x for x in waysToGetPeaks if not self.getProperty(x).isDefault]
-        if len(definedWaysToGetPeaks) == 0:
-            msg = "Purse peaks requires either a list of peaks, or ingredients to detect peaks"
-            errors["DetectorPeaks"] = msg
-            errors["DetectorPeakIngredients"] = msg
-        elif len(definedWaysToGetPeaks) == 2:
-            logger.warn(
-                """Both a list of detector peaks and ingredients were given;
-                the list will be used and ingredients ignored"""
-            )
```

Comment:
Note: this validation is no longer needed, as all peaks must come from the DetectorPeaks property